build/generate-go-licenses.go

```diff
@@ -82,6 +82,12 @@ func main() {
 		panic(err)
 	}
 
+	// Ensure file has a final newline
+	newline := []byte("\n")[0]
+	if jsonBytes[len(jsonBytes)-1] != newline {
+		jsonBytes = append(jsonBytes, newline)
+	}
```

Comment:
```suggestion
	if jsonBytes[len(jsonBytes)-1] != '\n' {
		jsonBytes = append(jsonBytes, '\n')
	}
```

Reply:
Huh, I didn't know you can compare slice with string, testing it...

Reply:
In Golang, `'\n'` is byte / rune, not string.